frontend/src/globals/unity-util.ts

```diff
@@ -674,6 +691,54 @@ export class UnityUtil {
 		}
 	}
 
+	/**
+	 * Called by the Calibration Tool when a user action changes the state of the Vector.
+	 * The line is given as a start position and end position in Project coordinates.
+	 */
+	/** @hidden */
+	public static calibrationVectorChanged(lineJson: string) {
+		const line = JSON.parse(lineJson);
+
+		// eslint-disable-next-line no-console
+		console.log(line);
```

Comment:
do we still need this?

If we want to keep this for debugging purpose I suggest we wrap it around `UnityUtil.verbose` like we do in other places